pkg/util/util.go

```diff
@@ -258,3 +260,17 @@ func GenerateSecureRandomString(n int) (string, error) {
 
 	return string(ret), nil
 }
+
+// EnsureKubeVirtGroupVersionKind ensures a provided object registered with KubeVirts generated schema
+// has GVK set correctly. This is required as client-go continues to return objects without
+// TypeMeta set as set out in the following issue: https://github.com/kubernetes/client-go/issues/413
+func EnsureKubeVirtGroupVersionKind(obj runtime.Object) (runtime.Object, error) {
```

Comment:
nit: I think that `Ensure` is a bit misleading. I think something like `Get..` or `Generate..` fits better here to emphasize that we're not performing assertions here, but rather generating a new object